CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### Improvements
 
+- Update USWDS from 3.4.1 to 3.5.0
+  - See release notes: https://github.com/uswds/uswds/releases/tag/v3.5.0
+  - Note "Breaking" changes in the above minor release. Additionally, the `inputPrefixSuffix` JavaScript named export has been removed.
```

Comment:
I'm very hesitant about shipping these without bumping the major version. It's complicated by the fact that USWDS ships breaking changes in minor releases 😬 

I'm sorta okay to bump the major version, but with the acknowledgment we'd probably be bumping it very frequently if most every USWDS is going to include changes like this.

Reply:
There's no limit to the numbers! We won't run out. Bump the big version

Reply:
Moved under "Breaking Changes" in rebased f591c80.